cdnauth/cdn.py

```diff
@@ -28,8 +28,18 @@ def session_jwt():
         if valid_redirect(request.args["cdn_resource"]) is False:
             logging.info(f"Invalid redirect URL: {request.args['cdn_resource']}")
             return "Invalid redirect URL"
+
+        cdn_resource = request.args["cdn_resource"]
+
+        if "timdexui=true" in cdn_resource:
+            timdexui = True
+        else:
+            timdexui = False
```

Comment:
I hate it! Mostly joking, but I feel like that takes more cognitive load to understand than the if/else option.

I'll give it some thought though, thanks for sharing a different approach. (I also avoid most ternary operators because they make me think too hard when trying to skim code)

Reply:
Fair points!  I'm not attached.  A pro for me is one less possibility for a variable name typo, but these IDEs and linters will hardly let that happen anymore.